### PR TITLE
Website updates

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

**Comment by asettle** [comment]
@bk201 @jillian-maroket - who is responsible for reviewing PRs here?

File: src/components/HomepageFeatures.js [hunk 1]
```diff
@@ -94,11 +94,11 @@ export default function HomepageFeatures() {
...
                                         <div>
                                             <h2 className="h3 font-weight-bolder">What is Harvester?</h2>
-                                            <p class="font-weight-light">Harvester is a modern Hyperconverged infrastructure (HCI) solution built for
-                                                bare metal servers using enterprise-grade open source technologies including Kubernetes, KubeVirt and
-                                                Longhorn. Designed for users looking for a cloud-native HCI solution, Harvester is a flexible and
-                                                affordable offering capable of putting VM workloads on the edge, close to your IoT, and integrated into
-                                                your cloud infrastructure.<br /></p>
+                                            <p class="font-weight-light">Harvester is a modern hyperconverged infrastructure (HCI) solution built for
+                                                bare metal servers using enterprise-grade open source technologies including Linux, KVM, Kubernetes, KubeVirt, and
```
Comment (jillian-maroket) [inline src/components/HomepageFeatures.js:98]:
```suggestion
                                                bare metal servers using enterprise-grade open-source technologies including Linux, KVM, Kubernetes, KubeVirt, and
```

File: src/components/HomepageFeatures.js [hunk 2]
```diff
@@ -94,11 +94,11 @@ export default function HomepageFeatures() {
...
+                                                Longhorn. Designed for users looking for a flexible and affordable solution to run cloud-native and
+                                                virtual machine (VM) workloads in your datacenter and at the edge, providing a single pane of glass for virtualization
```
Comment (jillian-maroket) [inline src/components/HomepageFeatures.js:100]:
```suggestion
                                                virtual machine (VM) workloads in your datacenter and at the edge, Harvester provides a single pane of glass for virtualization
```

Comment (jillian-maroket) [reply src/components/HomepageFeatures.js:100]: I converted the second half to a complete sentence.

**Review by jillian-maroket — COMMENTED** [review]
@asettle I made a few suggestions. I also have a question about the official tagline.

File: docusaurus.config.js [hunk 3]
```diff
@@ -130,7 +120,7 @@ const config = {
   customFields: {
     title: "Harvester - Open-source hyperconverged infrastructure",
     description:
-      "An open-source hyperconverged infrastructure (HCI) software for a cloud-native world",
+      "An open-source hyperconverged infrastructure solution for a cloud-native world",
```
Comment (asettle) [inline docusaurus.config.js:123]:
```suggestion
      The open-source hyperconverged infrastructure solution for a cloud-native world",

**Review by jillian-maroket — APPROVED** [review]

**Comment by asettle** [comment]
Closing and restarting to do this from one location - apologies. The same PR will appear soon.

**Pull request closed without merging**